### PR TITLE
Add homebrew installation instructions and linkify /docs/installation.

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Improve /docs/installation](https://trello.com/c/Z73XnUTp/1217-improve-docs-installation)

File: _development/installation.md [hunk 1]
```diff
@@ -13,24 +13,39 @@ Here are some of the highlights:
...
 
     $ curl -Ls https://convox.com/install/osx.zip > /tmp/convox.zip
     $ unzip /tmp/convox.zip -d /usr/local/bin
 
-#### Linux
+#### Homebrew
+
+Alternatively, on OSX you can also install via Homebrew:
+
+    $ brew install convox
```
Comment (mwarkentin) [inline _development/installation.md]: I noticed that equinox.io provides a homebrew tap automatically for releases: https://equinox.io/docs#brew

Not sure if it's worth looking into using that or just stick with the existing homebrew installation.

Comment (mattmanning) [inline _development/installation.md]: There seems to be an issue with `brew upgrade` we should add a note that `convox update` should be used instead.

Comment (soulshake) [inline _development/installation.md]: Thanks @mwarkentin, I made a note of it.
@mattmanning Fixed.

**Pull request merged**